storagegroup/storagegroup_test.go

```diff
@@ -179,7 +179,7 @@ func generateOIDList() []refs.ObjectID {
 	return mmV2
 }
 
-func TestStorageGroup_SetMembers_DoubleSetting(t *testing.T) {
+func TestStorageGroup_SetMembers_DoubleSetting(_ *testing.T) {
```

Comment:
You can use `require.NotPanics()` in the test body.

Reply:
done